cts/cts-cli.in

```diff
@@ -2771,7 +2771,7 @@ class AclsRegressionTest(RegressionTest):
                         expected_rc=ExitStatus.INSUFFICIENT_PRIV),
                 AclTest("niceguy: Replace - delete attribute (deny)",
                         "cibadmin --replace -p",
-                        setup="""cibadmin --replace --xml-text '<nvpair id="cib-bootstrap-options-enable-acl" name="enable-acl"/>'""",
+                        setup="""cibadmin --replace --xml-text '<primitive id="dummy" class="ocf" provider="pacemaker" type="Dummy"/>'""",
```

Comment:
@kgaillot @clumens Can one or both of you look at this and see if this is still a valid test of deleting an attribute? I'm *really* struggling to follow how these tests work under the new framework, with the setup hooks and whatnot. The "chris" tests don't look like they're doing what they're documented as doing, so I'm concerned that I'm misunderstanding pretty badly about what the original CIB vs. the attempted changes are.

Reply:
`AclTest` works by passing a CIB copy resulting from the `setup` command as stdin for the `cibadmin --replace` command. So, the test does testing creating/deleting an attribute, but ACLs apply at the element level, so the element ACL is what is really getting tested.

The new code here would be a sufficient test if the `dummy` primitive had an additional attribute (like `description`) that the replacement text doesn't.

Reply:
Thanks, both.

> The new code here would be a sufficient test if the `dummy` primitive had an additional attribute (like `description`) that the replacement text doesn't.

Right. I thought that it did, but I couldn't figure out *how*.

My confusion was because the input CIB gets printed *before* the "Begin test" message. So I thought it was the current CIB after the previous test.

I've split off the test change into its own commit and updated it so that it should be a valid test now. There's probably a fair amount of cleanup that could be done; now is not the time.